server/config/env/dev.js

```diff
@@ -3,5 +3,16 @@ module.exports = {
   port: 3001, // change with development port
   mongoUrl: "mongodb://localhost:27017/EMS",
   secret: "#$up3r$up3r$3cr3t#@@#",
-  tokenExpiry: "1h"
+  tokenExpiry: "1h",
```

Comment:
Add similar settings in prod.js file

Reply:
Done.

Reply:
tokenExpiry: "1h"  this is missing in prod .. @bhushan-oe  Is this intentional?